### DISCUSSION TITLE
Could Not Create Runner

### DISCUSSION THREAD

**Discussion by cyb3rzest** [projectdiscovery/subfinder#822] [Q&A]
**Describe the bug**
Last week, it was working fine, but from yesterday neither it is updating nor it is running.
always giving the same output `"[FTL] Could not create runner: file doesn't exist"`

**Subfinder version**
2.5.7

`subfinder -d domain.name -rL /path/to/resolvers -t 100 -o tmp/subfinder.lists`

**Screenshots**

```
               __    _____           __         
   _______  __/ /_  / __(_)___  ____/ /__  _____
  / ___/ / / / __ \/ /_/ / __ \/ __  / _ \/ ___/
 (__  ) /_/ / /_/ / __/ / / / / /_/ /  __/ /    
/____/\__,_/_.___/_/ /_/_/ /_/\__,_/\___/_/

                projectdiscovery.io

[INF] Current subfinder version v2.5.7 (latest)
[INF] Loading provider config from /root/.config/subfinder/provider-config.yaml
[FTL] Could not create runner: file doesn't exist
                                                                                                               
┌──(root💀ViTi-Security)-[~]
└─# 

```

**Comment by ehsandeep** [comment]
As per the error looks like `resolvers` file doesn't exist.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
